var/spack/repos/builtin/packages/py-fitter/package.py

```diff
@@ -12,20 +12,23 @@ class PyFitter(PythonPackage):
     most probable distribution and the best parameters."""
 
     homepage = "https://github.com/cokelaer/fitter"
-    pypi = "fitter/fitter-1.5.1.tar.gz"
+    pypi = "fitter/fitter-1.6.0.tar.gz"
 
     maintainers("carsonwoods")
 
     license("GPL-3.0-or-later")
 
+    version("1.6.0", sha256="908223d75b35d3846984bfb2fed1d2a926da5a30b95b704aa95b2a894227c0af")
+    version("1.5.2", sha256="afb33a8b1e24cdbc9318f55be72534e07028f25240ab76a4b081d27d1ed677d9")
     version("1.5.1", sha256="893b35ad0a84c3b96b63ec203a6a79effdba98777aed966ae61709f5e1e8cf99")
 
-    depends_on("py-setuptools", type="build")
-
-    depends_on("py-pandas", type=("build", "run"))
-    depends_on("py-numpy", type=("build", "run"))
-    depends_on("py-scipy@0.18:", type=("build", "run"))
-    depends_on("py-tqdm", type=("build", "run"))
-    depends_on("py-joblib", type=("build", "run"))
-    depends_on("py-click", type=("build", "run"))
-    depends_on("py-matplotlib", type=("build", "run"))
+    depends_on("py-poetry-core@1:", when="@1.6:", type="build")
+    depends_on("py-setuptools", when="@:1.6", type="build")
+    depends_on("python@3.8:3.9", type=("build", "run"))
+    depends_on("py-click@8.1.6:9", type=("build", "run"))
+    depends_on("py-joblib@1.3.1:2", type=("build", "run"))
```

Comment:
I think, given our version semantic, you mean:
```suggestion
    depends_on("py-joblib@1.3.1:1", type=("build", "run"))
```
i.e. `1.3.1 < x < 2` ? Similarly for other `depends_on` statements.

Reply:
Whoops, that's correct. I just forgot the proper semantics. Thanks!